packages/nillion/create-venv.sh

```diff
@@ -1,22 +1,35 @@
 #!/usr/bin/env bash
 
+PYTHON_MAJOR_MINOR_VERSION=""
+
+function check_python_version() {
+  required_version="3.11"
+  current_version=$(python3 --version | cut -d ' ' -f2)
+  PYTHON_MAJOR_MINOR_VERSION=$(echo $current_version | cut -d '.' -f1,2)
+  if ! printf '%s\n' "$required_version" "$current_version" | sort -V | head -n 1 | grep -qx "$required_version"; then
```

Comment:
Can you explain what's going on on this line because I'm wondering if it can simplified with `if [ "$required_version" != "$PYTHON_MAJOR_MINOR_VERSION" ]`?

Reply:
Oh nevermind. I think the error message below clarifies it. It's the "or higher" part that does it.